src/python/deepgnn/graph_engine/snark/tests/convert_test.py

```diff
@@ -488,7 +532,9 @@ def graph_with_sparse_features_json(folder):
             "node_type": 1,
             "node_weight": 1,
             "float_feature": {"0": [1], "1": [-0.03, -0.04]},
-            "sparse_float_feature": {"2": {"coordinates": [1, 3, 7], "values": [5.5]}},
+            "sparse_float_feature": {
+                "2": {"coordinates": [[1, 3, 7]], "values": [5.5]}
```

Comment:
I updated this test case from,
```python
  "sparse_float_feature": {"2": {"coordinates": [1, 3, 7], "values": [5.5]}},
```
to
```python
  "sparse_float_feature": {"2": {"coordinates": [[1, 3, 7]], "values": [5.5]}}
```
it seems like the original version was incorrect?

Reply:
we support both cases, a single list(in case of a single value) and a nested list. Your case is more explicit.

Reply:
Okay I reverted this.